tests/test_clinical.py

```diff
@@ -119,7 +119,7 @@ def test_clinicalPUT(self):
         self.assert200(rv)
         clinical_data = json.loads(rv.data)
         self.assertEquals(clinical_data['status'], 'unknown')
-        self.assertEquals(clinical_data['issued'], new_issued)
+        self.assertEquals(clinical_data['issued'][:-6], new_issued)  # chop tz
```

Comment:
Might be better to test that `clinical_data['issued'] == new_issued + "+00:00"`, in case we somehow aren't using UTC as the default TZ